ipaplatform/redhat/tasks.py

```diff
@@ -518,9 +514,9 @@ def _create_tmpfiles_dir(self, name, mode, uid, gid):
         os.chmod(name, mode)
         os.chown(name, uid, gid)
 
-    def create_tmpfiles_dirs(self):
+    def create_tmpfiles_dirs(self, ipaapi_user):
```

Comment:
How about you replace the code with ``systemd-tmpfiles`` and fold it into ``configure_tmpfiles``? You can get rid of duplicated configuration, too.

```
check_call(['systemd-tmpfiles', '--create', os.path.basename(ETC_TMPFILESD_IPA)]
```